bindings/node/lib/autoEncrypter.js

```diff
@@ -159,11 +159,18 @@ module.exports = function (modules) {
       // that we are not using the CSFLE shared library.
       if (!this._bypassMongocryptdAndCryptShared && !this.cryptSharedLibVersionInfo) {
         this._mongocryptdManager = new MongocryptdManager(options.extraOptions);
-        this._mongocryptdClient = new MongoClient(this._mongocryptdManager.uri, {
+        const clientOptions = {
           useNewUrlParser: true,
           useUnifiedTopology: true,
           serverSelectionTimeoutMS: 10000
-        });
+        };
+
+        const wasURIProvided = options.extraOptions && !!options.extraOptions.mongocryptdURI;
+        if (!wasURIProvided) {
```

Comment:
```suggestion
        if (options.extraOptions == null || typeof options.extraOptions.mongocryptdURI !== 'string')
```

Just a suggestion, just to read this as "there's no extra options, nor is there a URI string provided" so we set family to 4, but totally optional